manual/src/refman/extensions/attributes.etex

```diff
@@ -287,7 +287,7 @@ Some attributes are understood by the type-checker:
    "ocaml.local" or "local" take either "never", "always", "maybe" or
    nothing as payload on a function definition.  If no payload is
    provided, the default is "always".  The attribute controls an
-   optimization which consists in compiling a function into a static
+   optimization which consists of compiling a function into a static
```

Comment:
This sounds like a `consist in` to me?  The sentence can be reworded to `Compiling a function into a static continuation is the key feature of the optimization controlled by the optimization`

Reply:
reverted

Reply:
The other `consists in` ⇒ `consists of` fixes seemed fine to me: they were describing constituents. But reverting works too.